### PR TITLE
showed tokens of lp tokens

### PR DESCRIPTION
**Pull request by smartdevcom**

### CODE REVIEW

File: packages/ui/components/shared/CTokenIcon.tsx [hunk 1]
```diff
@@ -26,6 +29,9 @@ export const CTokenIcon = ({
           name={tokenData?.symbol ?? 'Loading...'}
           src={
             tokenData?.logoURL ||
+            `https://d1912tcoux65lj.cloudfront.net/token/${addressIcons[
```
[the remark this answers is not in the document]
Comment (smartdevcom) [reply packages/ui/components/shared/CTokenIcon.tsx:32]: image

Btw, the underlyings for `WBNB-DAI` and `BUSD-USDT` are same so hard to identity each other

[the remark this answers is not in the document]
Comment (carlomazzaferro) [reply packages/ui/components/shared/CTokenIcon.tsx:32]: > image
> 
> Btw, the underlyings for `WBNB-DAI` and `BUSD-USDT` are same so hard to identity each other

Hmmmm, I believe these are wrong. Let me check on this

[the remark this answers is not in the document]
Comment (smartdevcom) [reply packages/ui/components/shared/CTokenIcon.tsx:32]: image

Also `busd-usdt.png` exists in icon repo but couldn't find in URL https://d1912tcoux65lj.cloudfront.net/token/busd-usdt.png

**Comment by smartdevcom** [comment]
> Is this supposed to work yet? Looking at the code I can't tell how this should work just yet thinking
> 
> Screenshot 2022-06-07 at 14 21 48

image

**Pull request merged**